apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -1,8 +1,8 @@
 var React = require('react');
-var msg = require('@cdo/locale');
 var connect = require('react-redux').connect;
 
 var GameButtons = require('../templates/GameButtons').default;
+var CompletionButton = require('../templates/CompletionButton').default;
```

Comment:
would prefer import syntax for new code (even if that means this file has a mix of import/require)